build.sbt

```diff
@@ -4,17 +4,17 @@ lazy val buildSettings = Seq(
   organization := "com.github.finagle",
   version := "0.11.0-M4",
   scalaVersion := "2.11.8",
-  crossScalaVersions := Seq("2.10.6", "2.11.8")
+  crossScalaVersions := Seq("2.11.8", "2.12.0")
```

Comment:
but using sbt to 'crosscompile' with `> + package` doesn't work since there are (as mentioned, finagle) dependencies not yet available to use with 2.12.